bin/host/src/interop/local_kv.rs

```diff
@@ -37,14 +38,11 @@ impl KeyValueStore for LocalKeyValueStore {
             }
             L2_CLAIMED_POST_STATE_KEY => Some(self.cfg.claimed_l2_post_state.to_vec()),
             L2_CLAIMED_TIMESTAMP_KEY => Some(self.cfg.claimed_l2_timestamp.to_be_bytes().to_vec()),
-            L2_CHAIN_ID_KEY => Some(self.cfg.active_l2_chain_id().ok()?.to_be_bytes().to_vec()),
             L2_ROLLUP_CONFIG_KEY => {
                 let rollup_configs = self.cfg.read_rollup_configs().ok()?;
-                let active_rollup_config = rollup_configs
-                    .get(&self.cfg.active_l2_chain_id().ok()?)
-                    .cloned()
-                    .unwrap_or_default();
-                let serialized = serde_json::to_vec(&active_rollup_config).ok()?;
+                let serialized =
+                    serde_json::to_vec(&rollup_configs.into_iter().collect::<HashMap<_, _>>())
+                        .ok()?;
```

Comment:
Feels like this could also be squashed into a method on the `InteropHostCli`.

e.g. `pub fn config_vec(&self) -> Result<Vec<RollupConfig>, ...> { ... }`

Reply:
The `InteropHostCli` does already have `read_rollup_configs`, which returns a `HashMap` keying the L2 chain ID to the `RollupConfig`s.

That said I was originally trying to pass these as just an array of values - no need to iter -> collect again. Changed it back up to just serialize the map verbatim.